README.md

```diff
@@ -24,4 +23,4 @@ heroku config:set BOT_TOKEN=0123456789abcdef
 ```
 
 ## Author
-[Erik Boesen](https://github.com/ErikBoesen)
+[Will Shadow](https://github.com/thewilloftheshadow)
```

Comment:
I did write the original code so I would prefer that we both be listed here.

Reply:
:+1: true, that makes sense